lib/backend/dynamo/dynamodbbk.go

```diff
@@ -270,7 +271,18 @@ func New(ctx context.Context, params backend.Params) (*Backend, error) {
 		return nil, trace.Wrap(err)
 	}
 
-	var dynamoOpts []func(*dynamodb.Options)
+	dynamoResolver, err := endpoint.NewLoggingResolver[dynamodb.EndpointParameters](
```

Comment:
The type parameter is inferred through the type of the function argument:

```suggestion
	dynamoResolver, err := endpoint.NewLoggingResolver(
```